Rakefile

```diff
@@ -10,3 +10,20 @@ namespace :gem do
     system 'gem build trollolo.gemspec'
   end
 end
+
+# rake task for rspec and rubocop
+# rake -T (for rake tasks)
+require 'rubocop/rake_task'
+require 'rspec/core/rake_task'
+
+task default: %i[rubocop spec]
+
+desc 'Run tests'
```

Comment:
@rahul2240 the issue is about creating a single rake task that run both the tests and rubocop. 😉

Reply:
@Ana06 using rake will do both the tasks (test,style check) if there is some offence then only it will not complete.

Reply:
@rahul2240 what do you mean with `using rake`? what command would you run?

Reply:
using "rake" only will do both the task.

Reply:
@rahul2240 and what happens if we add another unrelated rake task in the future?

Reply:
@Ana06 Right now test and style check is set as default for rake command if unrelated rake task is added in future then we will have to use (rake taskname) to run it.

Reply:
@Ana06 If I am doing it wrong please let me know I am ready to start again :smile:

Reply:
@rahul2240 if we add a task in the future you will have to run rake taskname and then you can not run both things at the same time. We want a single rake task, not two, that runs everything that Travis run.

Reply:
@Ana06 Ok now I got it. Cann't we add that task to default? so that it runs automatically with rake command, Or we need a more simpler method?